charon/src/bin/charon-driver/translate/translate_constants.rs

```diff
@@ -187,8 +191,8 @@ impl<'tcx, 'ctx> BodyTransCtx<'tcx, 'ctx> {
             }
             ConstantExprKind::Memory(bytes) => RawConstantExpr::RawMemory(bytes.clone()),
             ConstantExprKind::Todo(msg) => {
-                // Case not yet handled by hax
-                raise_error!(self, span, "Unsupported constant: {:?}", msg)
+                register_error!(self, span, "Unsupported constant by Hax: {:?}", msg);
+                RawConstantExpr::Opaque("Hax: ".to_owned() + msg)
```

Comment:
We should not mention hax to end-users if we can avoid it.
```suggestion
                register_error!(self, span, "Unsupported constant: {:?}", msg);
                RawConstantExpr::Opaque(msg)
```